zerver/lib/events.py

```diff
@@ -232,7 +232,7 @@ def fetch_initial_state_data(
         ] = realm.message_content_delete_limit_seconds
         state[
             "realm_community_topic_editing_limit_seconds"
-        ] = Realm.DEFAULT_COMMUNITY_TOPIC_EDITING_LIMIT_SECONDS
+        ] = Realm.DEFAULT_TOPIC_EDITING_LIMIT_SECONDS
 
```

Comment:
Can also rename `realm_community_topic_editing_limit_seconds`, but wanted to make sure that we are on page with other clients using this.

Reply:
I'd check with other clients; that field is new and it's possible nobody is using it yet.

Reply:
We have decided [here](https://chat.zulip.org/#narrow/stream/378-api-design/topic/allow_community_topic_editing/near/1200079) that we should not change this.